torchaudio/backend/sox_io_backend.py

```diff
@@ -9,20 +9,27 @@
 
 
 @_mod_utils.requires_module('torchaudio._torchaudio')
-def info(filepath: str) -> AudioMetaData:
+def info(
+        filepath: str,
+        format: Optional[str] = None,
+) -> AudioMetaData:
     """Get signal information of an audio file.
 
     Args:
         filepath (str or pathlib.Path):
             Path to audio file. This function also handles ``pathlib.Path`` objects,
             but is annotated as ``str`` for TorchScript compatibility.
+        format (str, optional):
+            Hint libsox about the audio format. This argument helps libsox detect the audio
```

Comment:
Verify if this is fallback or overwrite, and update the docstring.

Reply:
Okay, it's overwrite. https://github.com/dmkrepo/libsox/blob/master/src/formats.c#L475-L481